web/src/App.tsx

```diff
@@ -18,9 +19,7 @@ function App() {
       <div className="card">
         <IncrementButton incrementBy={1} />
         <IncrementButton incrementBy={2} />
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
+        <CanvasBoard height={200} width={400} />
```

Comment:
do 200 & 400 need to be in curlys here?

Reply:
![image](https://user-images.githubusercontent.com/85482523/198069194-3c385b4f-daac-489c-9957-76f3daf3690f.png)
Yes